code/modules/projectiles/projectile.dm

```diff
@@ -719,7 +719,7 @@ So if we are on the 32th absolute pixel coordinate we are on tile 1, but if we a
 
 
 /mob/living/carbon/human/projectile_hit(obj/projectile/proj, cardinal_move, uncrossing)
-	if(proj.ammo.flags_ammo_behavior & AMMO_SKIPS_HUMANS && get_target_lock(proj.ammo.iff_signal))
+	if(get_target_lock(proj.ammo.iff_signal))
 		return FALSE
```

Comment:
Why

Reply:
reverted to original if check and added or check for aim mode